ramp_custom_lwfa/predictions.py

```
@@ -38,5 +38,22 @@ def combine(cls, predictions_list, index_list=None):
                 filtered_predictions = combined_predictions
 
             return cls(y_pred=np.array(filtered_predictions, dtype=object))
+        
+        @property
+        def valid_indexes(self):
+            """Return valid indexes, handling empty predictions."""
+            try:
+                if len(self.y_pred.shape) > 1:
+                    # Handle case where shape is (n,0)
+                    return np.ones(self.y_pred.shape[0], dtype=bool)
```

what if a student send something `n, 1`? This should raise an error